maquetado/scss/styles.scss

```diff
@@ -0,0 +1,17 @@
+.title {
+  color: red;
+}
+.titles {
+  display: flex;
+}
+
+img {
```

Comment:
Isn't a good practice to assign general styles to the elements. Also you should try with `pseudo elements` of CSS and improve this